### PR TITLE
Governance Game Documentation

### PR DESCRIPTION
**Pull request by kasunkavinda**
Governance Game Documentation

### CODE REVIEW

File: governance-game.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+# Governance Game
+
```
Comment (wojake) [inline governance-game.md:2]:
```suggestion

## Sashimono

```

Comment (wojake) [reply governance-game.md:2]: Fixed

File: governance-game.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+## Participants
+
+There are two classes of Participant in the Governance Game.
+
+- The Foundation: The Foundation always has 1 vote and has special rights.
+- Valid Hosts: Accounts that hold a registration token for the previous 3 continuous months and are not eligible to be pruned due to unreliability.
+
+## Governance
+
+Participant can submit a Proposal with new Hook hashes of above mentioned 3 Hooks. Once that Proposal is continuously supported by 80% of eligible Participants for 2 weeks, the Hook will be changed accordingly.
```
Comment (ravinsp) [inline governance-game.md:22]: the Hook will be changed accordingly. --> The existing hooks will be replaced by the proposed hooks.

File: governance-game.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+### Submitting a proposal
+
+- A Proposal represents the hashes <governance_hooks_hash><registry_hook_hash><heartbeat_hook_hash> of the new Hook to replace an existing Hook.
```
Comment (ravinsp) [inline governance-game.md:26]: Additional point to mention: The hooks which bears the proposed hashes must be deployed to some existing XRPL account.

File: governance-game.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+- If a Proposal has not Succeeded three months after being proposed, it will be purged. If a Proposal expires, the Proposer loses all their staked EVRs.
+- Lost EVRs are added to that Epoch’s reward pool.
+
+### Voting
+
+- Participants vote is captured via their heartbeat.
+- They either Support or Reject a Proposal.
+  - Reject is the default.
+  - Support is positive vote for the Proposal.
```
Comment (ravinsp) [inline governance-game.md:47]: Let's mention that hosts can make their choice of voting via evernode cli.

File: governance-game.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+- Lost EVRs are added to that Epoch’s reward pool.
+
+### Purging a Proposal
+
+- If a Proposal has not Succeeded three months after being proposed, it will be purged. If a Proposal expires, the Proposer loses all their staked EVRs.
+- Lost EVRs are added to that Epoch’s reward pool.
+
+### Voting
+
+- Participants vote is captured via their heartbeat.
```
Comment (ravinsp) [inline governance-game.md:44]: Participants vote is captured via their heartbeat, which is managed by Evernode software installed on the host.

**Review by ravinsp — APPROVED** [review]

**Review by chalith — APPROVED** [review]

**Review by chalith — APPROVED** [review]

**Pull request merged**